desktop-src/Debug/symbol-paths.md

```diff
@@ -60,37 +61,30 @@ The symbol handler treats a path element as a cache element if it begins with th
 
 To see how this works, set this search path.
 
-<dl> **.;cache\****myCache***;srv\*\\\\symbols\\symbols**  
-</dl>
+`**.;cache\****myCache***;srv\*\\\\symbols\\symbols**`
```

Comment:
Should there be so many stars and so many backslashes? Looks like some over-escaping